app.authpass.AuthPass.yaml

```diff
@@ -0,0 +1,61 @@
+app-id: app.authpass.AuthPass
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+command: authpass
+finish-args:
+  # Filesystem and network access
+  - --filesystem=home
```

Comment:
I think while people can really use any location(s) for the databases, constraining this a little bit more to `Documents` and/or `Downloads` might be better.

Reply:
It's a Flutter app so I don't really have hopes for it to support XDG properly.

But maybe it needs a `persist` for the settings directory.

Reply:
right now it stores 
* app settings into `$HOME/.authpass/`
* Cached files from cloud storage into `$TMPDIR`
* "internal" password files go into `$XDG_DATA_HOME` by default. But the user can obviously open a file at any location using a native open file dialog.

Reply:
> app settings into $HOME/.authpass/

settings don't belong tehere, even outside a Flatpak. Please use $XDG_CONFIG_HOME.

> Cached files from cloud storage into $TMPDIR

Note that $TMPDIR can be removed after a Flatpak is closed. If you want to use the cache longer than the App runs, use $XDG_CACHE_HOME. Otherwise $TMPDIR is the right choice.

Reply:
@JakobDev okay I've implemented the changes.. https://github.com/authpass/authpass/commit/be25e8d05a8e3a18e56d21522118c7f892feae74#diff-f966a0c220ffee66c7b945faa18a71acd3f323788477a7ca483721e33572e171R59-R62

but it would be nice if we could still get this finished before I create a new release.